hips/draw/tests/test_simple.py

```diff
@@ -1,102 +1,79 @@
-    # The same example is used in the high level docs getting started page
-    url = 'http://alasky.unistra.fr/DSS/DSS2Merged/properties'
-    hips_survey = HipsSurveyProperties.fetch(url)
-    geometry = make_test_wcs_geometry(case=2)
-    data = make_sky_image(geometry=geometry, hips_survey=hips_survey, tile_format='fits')
-    assert data.shape == geometry.shape
-    assert data.dtype == np.float64
-
-    assert_allclose(np.sum(data), 8757489268.044867)
-    assert_allclose(data[200, 994], 2213.30874796)
-    assert_allclose(data[200, 995], 2296.93885940)
-
-hips_order_pars = [
-    dict(order=7, fov="3 deg"),
-    dict(order=5, fov="10 deg"),
-    dict(order=4, fov="15 deg"),
-]
-
-
-@requires_hips_extra()
-@pytest.mark.parametrize('pars', hips_order_pars)
-def test_compute_matching_hips_order(pars):
-    full_path = get_hips_extra_file('datasets/samples/2MASS6XH/properties')
-    hips_survey = HipsSurveyProperties.read(filename=full_path)
-    geometry = WCSGeometry.create_simple(
-        skydir=SkyCoord(0, 0, unit='deg', frame='icrs'),
-        width=2000, height=1000, fov=pars['fov'],
-        coordsys='icrs', projection='AIT'
-    )
-    assert compute_matching_hips_order(geometry, hips_survey) == pars['order']
-
-get_hips_order_for_resolution_pars = [
-    dict(tile_width=512, resolution=0.01232, resolution_res=0.06395791924665553, order=4),
-    dict(tile_width=256, resolution=0.0016022, resolution_res=0.003997369952915971, order=8),
-    dict(tile_width=128, resolution=0.00009032, resolution_res=0.00012491781102862408, order=13),
-]
-
-
-@pytest.mark.parametrize('pars', get_hips_order_for_resolution_pars)
-def test_get_hips_order_for_resolution(pars):
-    hips_order = _get_hips_order_for_resolution(pars['tile_width'], pars['resolution'])
-    assert hips_order == pars['order']
-    hips_resolution = hp.nside2resol(hp.order2nside(hips_order))
-    assert_allclose(hips_resolution, pars['resolution_res'])
-# TODO: add tests for SimpleTilePainter with asserts on the intermediate computed things.
+    painter = make_sky_image(geometry=geometry, hips_survey=hips_survey, tile_format=pars['file_format'])
+    assert painter.image.shape == pars['shape']
+    assert painter.image.dtype == np.float64
+    assert_allclose(np.sum(painter.image), pars['data_sum'])
+    assert_allclose(painter.image[200, 994], pars['data_1'])
+    assert_allclose(painter.image[200, 995], pars['data_2'])
+
+
+class TestSimpleTilePainter:
+    @classmethod
+    @remote_data
+    def setup_class(cls):
+        url = 'http://alasky.unistra.fr/DSS/DSS2Merged/properties'
+        cls.hips_survey = HipsSurveyProperties.fetch(url)
+        cls.geometry = WCSGeometry.create_simple(
+            skydir=SkyCoord(0, 0, unit='deg', frame='icrs'),
+            width=2000, height=1000, fov="3 deg",
+            coordsys='icrs', projection='AIT'
+        )
+        cls.simple_tile_painter = SimpleTilePainter(cls.geometry, cls.hips_survey, 'fits')
+
+    @remote_data
```

Comment:
At the moment you added `@remote_data` on every single method of `TestSimpleTilePainter`.
Please remove it from all the methods, and only put it on `TestSimpleTilePainter` instead.
(and then later we'll try to change the code / tests so that most tests don't require remote data, but not now.)